recipes/blast/conda_build_config.yaml

```diff
@@ -1,6 +1,6 @@
 # Clang version pin for RpsbProc. Clang>14 will raise:
 # "conversion from 'const *_t' (aka 'const unsigned long long') to 'const JSVar' is ambiguous"
 cxx_compiler_version:  # [osx]
-  - 14  # [osx]
+  - 15  # [osx]
```

Comment:
Isn't osx default compiler version 16 by now?

Reply:
Indeed: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/bc1bd54181efe8c52967a8599b0d45659ed1df83/recipe/conda_build_config.yaml#L8

you can probably get rid of the build config file as a result!

Reply:
Ah I see, sorry, clang >14 raises errors, got it

Reply:
I'm just trying some stuff out. But it may also be an option to take out RpsbProc completely and putting it in its own recipe if it doesn't want to play nice with Clang16